packages/esm-patient-medications-app/src/api/api.ts

```diff
@@ -12,15 +12,15 @@ import { DrugOrderBasketItem } from '../types';
  * @param status Allows fetching either all orders or only active orders.
  */
 export function usePatientOrders(patientUuid: string, status: 'ACTIVE' | 'any') {
-  const { careSettingUuid, drugOrderTypeUUID } = useConfig() as ConfigObject;
+  const { drugOrderTypeUUID } = useConfig() as ConfigObject;
   const customRepresentation =
     'custom:(uuid,dosingType,orderNumber,accessionNumber,' +
-    'patient:ref,action,careSetting:ref,previousOrder:ref,dateActivated,scheduledDate,dateStopped,autoExpireDate,' +
```

Comment:
@ibacher, is there any utility in keeping this `careSetting:ref` bit in the custom representation?

Reply:
I see. Does everything else look OK in this PR then?

Reply:
So, we should use the care setting when creating orders, just not displaying them. I think it's ok to hard-code for now.

Reply:
Can you do what's required, @Jexsie?

Reply:
updated. Thanks @ibacher  @denniskigen